HISTORY.rst

```diff
@@ -2,9 +2,19 @@
 History
 =======
 
+0.2.0 (2025-06-23)
+-------------------
+
+* Allow overriding the entire cap_app_name of a one-click app (#14)
+* BREAKING: Drop namespace parameter for one-click app (#14)
+* Auto-assign tag when deploying one-click app (#14)
+* Override one-click app tag on deploy_one_click_app & update_app (#20)
+* Add projects support (#18)
+
 0.1.24 (2024-12-16)
 -------------------
 
+* Support "command" in service_update_override for one-click apps (#13)
```

Comment:
This change made it into 0.1.24 but was missed from the changelog.  Probably not a big deal.